pkgs/development/rocm-modules/6/default.nix

```diff
@@ -4,7 +4,7 @@
 , symlinkJoin
 , fetchFromGitHub
 , cudaPackages
-, python3Packages
```

Comment:
I would recommend to overwrite python3Packages with python311Packages in this file, instead of every other file.

Reply:
> I would recommend to overwrite python3Packages with python311Packages in this file, instead of every other file.

I was thinking about it, but it turned out to be a bit harder then running a `sed` command, because there is no `newScope` in this file.. I'm open to suggestions though.